action.yml

```diff
@@ -1,33 +1,42 @@
 name: 'Okteto Test'
-description: 'Run tests in okteto'
+description: 'Run tests in Okteto manifest'
```

Comment:
is this intentional?

Reply:
Yes, it was not clear if it was in the okteto platform or using an okteto manifest so I decided to update it